ui/partials/admin/exclusions_list.html

```diff
@@ -3,8 +3,8 @@
        ng-controller="ExclusionsListCtrl"
        ng-init="populateExclusionsData()">
     <button ng-click="go('exclusions_detail', {id: 'new'})" type="button"
-            ng-disabled="!user.is_staff"
```

Comment:
I think we should either hold off the s/is_staff/isSheriff/ until after this bug, or do so in an earlier PR to make the diff easier to read.